### DISCUSSION TITLE
Issue with opening modal when selecting checkbox for bulk actions in clickable rows

### DISCUSSION THREAD

**Discussion by slossetti** [rappasoft/laravel-livewire-tables#1947] [Q&A] [answered]
Hello everyone!,

I added to make the rows clickable:

` $this->setTrAttributes(function ($row) {
                return [
                    'wire:click' => '$dispatch("showModal", { id: ' . $row->id . ' })',
                    'class' => 'cursor-pointer hover:bg-yellow-50 dark:hover:bg-gray-900',
                ];
            });`
            
The issue I'm facing is that when I select a checkbox to perform bulk actions, the modal opens unintentionally. I would like to prevent the modal from opening when the checkbox is selected.

Could you assist me in resolving this issue?

Thank you!

Best regards,
Sergio.

**Comment by lrljoe** [comment, marked as answer]
As you're using AlpineJS, any click on the "tr" element will result in wire:click being fired.

If you add this to your configure(), then it should ensure that the checkbox doesn't interact with your "tr" wide on-click

```php
    public function configure(): void
    {

      $this->setBulkActionsTdCheckboxAttributes([
          'x-on:click.stop' => '',
          'default' => true
    ]);
```

**Reply by slossetti** [reply]
Thanks!! I didn't know about that function. Another question, is there a way to make the marked rows in another color?

**Reply by lrljoe** [reply]
If you're familiar enough with AlpineJS - then yes, absolutely.  Otherwise it'll be a bit difficult for me to explain on GH, and something I don't necessarily want to end up supporting long-term, do feel free to reach out on Discord if you want pointers on how to achieve it unofficially tho!

**Reply by slossetti** [reply]
Thanks!. I will try using alpine. I'm not an expert but I will try.
